fetch.bs

```diff
@@ -5061,6 +5061,13 @@ optional boolean <var>forceNewConnection</var> (default false), run these steps:
     <ul>
      <li><p>Follow the relevant requirements from HTTP. [[!HTTP]] [[!HTTP-SEMANTICS]] [[!HTTP-COND]] [[!HTTP-CACHING]] [[!HTTP-AUTH]]
 
+     <li><p> In resolving the HTTP request, there is an evitable situaion that the user agent must
+     resend the <var>request</var> body (e.g. socket time-out).
+     If <var>request</var>'s <a for=request>body</a> is non-null, and <var>request</var>'s
+      <a for=request>body</a>'s <a for=body>source</a> is null, there is an <a>implementation-defined</a>
+      cache limit not to cache the entire <a for=request>body</a>'s stream. If the cache runs out,
+      return a <a>network error</a>.
```

Comment:
This paragraph sounds like a mixture of normative text and non-normative notes. How about the following?

The user agent may have a buffer of an implementation-defined size and store the part of request's body to the buffer, when request's body is non-null and its source is null. The user agent may use the buffer to re-send request's body to the network.

(note) The re-sending is needed when the connection is timed out, for example. If request's body is sent more than the buffer size, it leads to a network error.

(note) The buffer is not needed when request's body's source is non-null, because the body can be re-created from request's body's source.

Reply:
Done.